docs/root/dev/writing-intel-modules.md

```diff
@@ -83,7 +82,7 @@ def load_emr_clusters(
 ```
 
 
-`load_graph_data()` requires an `ingestion_query` to be generated from `CartographyNodeSchema` and `CartographyRelSchema` objects. [cartography.graph.querybuilder.build_ingestion_query()](https://github.com/lyft/cartography/blob/e6ada9a1a741b83a34c1c3207515a1863debeeb9/cartography/graph/querybuilder.py#L312) does just that: it accepts those schema objects as input and returns a well-formed and optimized cypher query.
```

Comment:
Modules won't be directly calling `load_graph_data` anymore?

Reply:
Correct, I updated the doc. They will be calling `load()`. It's early enough to make a change like this before more modules start using it